.github/workflows/prod.yml

```diff
@@ -1,17 +1,12 @@
-name: PROD (release)
+name: PROD (dispatch)
 
 on:
-  release:
-    types: [published]
   workflow_dispatch:
     inputs:
       tag:
-        description: Container tag; usually prod or PR number, can be omitted
-        type: string
-  workflow_call:
-    inputs:
-      tag:
-        description: Container tag; usually prod or PR number, can be omitted
+        description: Container tag; usually test, prod or PR number
+        default: test
```

Comment:
Sorry, just for my confusion dumb question:
If for new release we use 'test', correct?
If for some reason, to back to previous release, using 'prod' (previous prod release) is it? or what's 'prod'?

Reply:
Yes, test.  It's also the default so you don't actually have to enter anything.  Prod and prod-backup are there too for redundancy.  Hopefully we won't ever need them.